cmd/templatizer/app/flags.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Options represent configurable parameters for jaeger-templatizer
+type Options struct {
+	Mapping   string
+	EsVersion int64
+	Shards    int64
+	Replicas  int64
+	EsPrefix  string
+	UseILM    string // using string as util is being used in python and using bool leads to type issues.
+}
+
+const (
+	mappingFlag   = "mapping"
+	esVersionFlag = "esVersion"
+	shardsFlag    = "shards"
+	replicasFlag  = "replicas"
+	esPrefixFlag  = "esPrefix"
```

Comment:
esPrefix is an ambiguous name, can we call it `index-prefix`?

Reply:
Have taken this convention from existing python code. Wanted to be consistent, let me know if you want me  change this in python scripts as well.